formsg/webhook.py

```diff
@@ -44,11 +45,7 @@ def verify_signature(webhook_uri, signature_header, signature_expiry_seconds=60)
 
 
 def decrypt_content(body_json, secret_key):
-    if 'data' in body_json:
-        encrypted_content = body_json['data']['encryptedContent']
-    else:
-        encrypted_content = body_json['encryptedContent']  # old version POST body
-    #end if
+    encrypted_content = get_data_from_json(body_json)['encryptedContent']
```

Comment:
We can just do `body_json.get('data', body_json)`

Reply:
Might want to leave a comment about this behavior for new and old formsg incoming webhook payloads.

> Some FormSG submissions are in a `data` field while others are not. Different formsg versions perhaps?